src/operations/eval.ts

```diff
@@ -5,16 +5,15 @@ import { MongoServerError } from '../error';
 import { ReadPreference } from '../read_preference';
 import type { Server } from '../sdam/server';
 import type { ClientSession } from '../sessions';
-import type { Callback } from '../utils';
-import { CommandCallbackOperation, type CommandOperationOptions } from './command';
+import { CommandOperation, type CommandOperationOptions } from './command';
 
 /** @public */
 export interface EvalOptions extends CommandOperationOptions {
```

Comment:
This operation is unused but because these are public APIs we have to make sure the change is visible in the changelog. Can you file a ticket to remove the EvalOperation and related APIs.

But we can keep the changes we're making here so that we can still remove the callback super class.